<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="8" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-06T15:28:16 IST" name="Suite" finished-at="2025-04-06T15:28:40 IST" duration-ms="24453">
    <groups>
    </groups>
    <test started-at="2025-04-06T15:28:16 IST" name="Test" finished-at="2025-04-06T15:28:40 IST" duration-ms="24453">
      <class name="api.testcases.TestUserFlowWithPojo">
        <test-method is-config="true" signature="setUp()[pri:0, instance:api.testcases.TestUserFlowWithPojo@2e222612]" started-at="2025-04-06T15:28:16 IST" name="setUp" finished-at="2025-04-06T15:28:17 IST" duration-ms="418" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testUserCreation()[pri:1, instance:api.testcases.TestUserFlowWithPojo@2e222612]" started-at="2025-04-06T15:28:17 IST" name="testUserCreation" finished-at="2025-04-06T15:28:21 IST" duration-ms="4422" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserCreation -->
        <test-method signature="testCreatedUser()[pri:2, instance:api.testcases.TestUserFlowWithPojo@2e222612]" started-at="2025-04-06T15:28:21 IST" name="testCreatedUser" finished-at="2025-04-06T15:28:22 IST" duration-ms="1129" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatedUser -->
        <test-method signature="testUserUpdate()[pri:3, instance:api.testcases.TestUserFlowWithPojo@2e222612]" started-at="2025-04-06T15:28:22 IST" name="testUserUpdate" finished-at="2025-04-06T15:28:24 IST" duration-ms="2183" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at api.testcases.TestUserFlowWithPojo.testUserUpdate(TestUserFlowWithPojo.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserUpdate -->
        <test-method signature="testUserDeletion()[pri:5, instance:api.testcases.TestUserFlowWithPojo@2e222612]" started-at="2025-04-06T15:28:24 IST" name="testUserDeletion" finished-at="2025-04-06T15:28:26 IST" duration-ms="1121" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserDeletion -->
      </class> <!-- api.testcases.TestUserFlowWithPojo -->
      <class name="api.testcases.TestUserFlowWithDataProviderExcelData">
        <test-method signature="testUserCreation(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:api.testcases.TestUserFlowWithDataProviderExcelData@34f7cfd9]" started-at="2025-04-06T15:28:27 IST" name="testUserCreation" data-provider="Data" finished-at="2025-04-06T15:28:29 IST" duration-ms="1037" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1010]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[John]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Canedy]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[a@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@1]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567897]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserCreation -->
        <test-method signature="testUserCreation(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:api.testcases.TestUserFlowWithDataProviderExcelData@34f7cfd9]" started-at="2025-04-06T15:28:29 IST" name="testUserCreation" data-provider="Data" finished-at="2025-04-06T15:28:30 IST" duration-ms="1031" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1020]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Kim]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Kom]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[b@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@2]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567897]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserCreation -->
        <test-method signature="testUserCreation(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:api.testcases.TestUserFlowWithDataProviderExcelData@34f7cfd9]" started-at="2025-04-06T15:28:30 IST" name="testUserCreation" data-provider="Data" finished-at="2025-04-06T15:28:31 IST" duration-ms="1102" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1030]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[testuser3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Smith]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Steve]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[c@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@3]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567897]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserCreation -->
        <test-method signature="testDeleteUser(java.lang.String)[pri:2, instance:api.testcases.TestUserFlowWithDataProviderExcelData@34f7cfd9]" started-at="2025-04-06T15:28:31 IST" name="testDeleteUser" data-provider="UserNames" finished-at="2025-04-06T15:28:34 IST" duration-ms="3033" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUser -->
        <test-method signature="testDeleteUser(java.lang.String)[pri:2, instance:api.testcases.TestUserFlowWithDataProviderExcelData@34f7cfd9]" started-at="2025-04-06T15:28:34 IST" name="testDeleteUser" data-provider="UserNames" finished-at="2025-04-06T15:28:37 IST" duration-ms="3078" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at api.testcases.TestUserFlowWithDataProviderExcelData.testDeleteUser(TestUserFlowWithDataProviderExcelData.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUser -->
        <test-method signature="testDeleteUser(java.lang.String)[pri:2, instance:api.testcases.TestUserFlowWithDataProviderExcelData@34f7cfd9]" started-at="2025-04-06T15:28:37 IST" name="testDeleteUser" data-provider="UserNames" finished-at="2025-04-06T15:28:40 IST" duration-ms="3087" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUser -->
      </class> <!-- api.testcases.TestUserFlowWithDataProviderExcelData -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
